security-analysis/security-analysis-api/src/main/java/com/powsybl/security/json/OperatorStrategyListSerializer.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.json;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+import com.powsybl.security.operator.strategy.OperatorStrategyList;
+
+import java.io.IOException;
+
+import static com.powsybl.security.operator.strategy.OperatorStrategyList.VERSION;
+
+/**
+ * @author Etienne Lesot <etienne.lesot@rte-france.com>
+ */
+public class OperatorStrategyListSerializer extends StdSerializer<OperatorStrategyList> {
+
+    protected OperatorStrategyListSerializer() {
```

Comment:
we should probably let the serializer/deserializers constructors public, in case it needs to be used by third-parties, although in most cases only the JSON module will be needed